docs/troubleshooting.adoc

```diff
@@ -0,0 +1,157 @@
+= Troubleshooting Landrush
+:toc:
+:toc-placement!:
+
+Use this document to identify and resolve basic Landrush issues that you encounter.
+https://github.com/vagrant-landrush/landrush#landrush-dns-for-vagrant-[Landrush]
+is a platform-agnostic DNS server used by ADB to provide name based access to
+services, such as Openshift, running in the ADB.
+
+As an ADB user, you download Landrush as a vagrant plugin. For further details
+on downloading Landrush, refer to the
+link:../installing.rst[Installation document].
+
+'''
+toc::[]
+'''
+
+==== I am unable to access the Internet or override the Landrush upstream DNS?
+By default, Landrush uses Google's DNS server, with the IP address 8.8.8.8, to
+resolve DNS queries that are not for Landrush managed hostnames. If your network
+blocks access to the Google DNS servers, due to firewalls, VPN and so on, as a workaround,
+you need to set a custom upstream DNS server. This server is configured with the
+`config.landrush.upstream` option.
+
+For example, to use 10.25.35.4 as a DNS server, add this line to your `Vagrantfile`:
+
+`config.landrush.upstream ‘10.25.35.4’`
```

Comment:
Real sharp eyes @budhrg those were single quotes but relics from the google doc. Changed them.